test/app/submissions_test.rb

```diff
@@ -43,7 +43,7 @@ def assert_response_status(expected_status)
   end
 
   def test_guests_can_view_submissions
-    Attempt.new(alice, Iteration.new('word-count/file.rb' => 'CODE')).save
+    Attempt.new(alice, Iteration.new('ruby/file.rb' => 'CODE')).save
```

Comment:
So is this change okay then? It maybe a bit confusing though when reading the whole test because the developer sees that only this part of the whole test file uses the `ruby` slug.